src/content/app-platform/apps/observability/_index.md

```diff
@@ -0,0 +1,43 @@
+---
+linkTitle: Observability Stack
+title: Managed Observability Stack (Monitoring, Logging, Service Mesh)
+description: Overview of the Giant Swarm managed observability stack of apps, which includes monitoring, logging, and service mesh
+weight: 10
+menu:
+  main:
+    parent: app-platform-apps
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-halo
+user_questions:
+  - What app do you recommend for monitoring?
+  - What app do you recommend for logging?
+  - What app do you recommend for service mesh?
+last_review_date: 2021-08-18
+---
+
+# Giant Swarm managed Observability Stack of Apps
+
+Observability is based on three main data sources: logs, metrics and tracing*. To cover these needs Giant Swarm provides its customers with a managed observability stack. The apps included in this stack are listed below. We chose these apps based on our general principles of preferring open source over proprietary solutions, as well as simplicity, performance, and speed over complexity.
```

Comment:
The `*` here should be escaped at least

Reply:
Maybe @marians has a good idea how to do this better in markdown. This leads to additional information. Maybe add a link or something? Or put the text from the botton in a `>` quote directly below this block?